### DISCUSSION TITLE
Check to see location is with a polygon on a layer

### DISCUSSION THREAD

**Discussion by EmmittS22** [Mapsui/Mapsui#2373] [Q&A]
I'm new to mapsui and maui dev.
Looking for info on Mapsui.Geometries.Utilities.Algorithms.PointInPolygon method and Mapsui.Geometries.Polygon.Contains method seen these mentioned in  web search,  ideally looking to loop through polygons on a layer and see if the  device location is within the polygon

**Comment by pauldendulk** [comment]
In Mapsui v4 we replaced the Mapsui.Geometries with those of NTS (which is a big improvement). If you want to now more about those kind of operations you should look in the [documentation of NTS itself](https://github.com/NetTopologySuite/NetTopologySuite?tab=readme-ov-file).

Yet, if you just want to know if a polygon is hit in the map you can use the Info event. Go the the [online samples](https://mapsui.com/samples/). Select "Demo" category and "1 Map Info" sample. The 'source code' tab shows you how to build that.
